src/lib/post.ts

```diff
@@ -98,5 +100,26 @@ const getPostByTag = (tag: string, fields: Field[] = POST_FIELDS): Post[] => {
   return posts.filter((post) => post.tags.includes(tag));
 };
 
-export { getPostBySlug, getAllPosts, getPostByTag };
+const randomPostSlug = (excludedSlugs: string[] = []): string | null => {
+  const slugs = getPostSlugs().filter((slug) => !excludedSlugs.includes(slug));
```

Comment:
If I understand the implementation correctly, this one excludes the savedSlugs from all the slugs present, making them newSlugs, do you think we can rename it to `newSlugs`?

```suggestion
  const newSlugs = getPostSlugs().filter((slug) => !excludedSlugs.includes(slug));
```

Reply:
Fixed in https://github.com/nimblehq/til-web/pull/33/commits/0a56f2f3b162d6d38fe44642fba0a52fe5a50856